Docs: Update/Refresh OneLogin SSO guide (#28444)
gravitational/teleport: docs/pages/access-controls/sso/one-login.mdx

---

> title: Teleport Authentication with One Login as an SSO provider

Reviewer: ```suggestion
title: Teleport Authentication with OneLogin as an SSO Provider
```
- Using title case per the style guide
- Using the official product spelling

> description: How to configure Teleport access using One Login as an SSO provider

Reviewer: We should `s/One Login/OneLogin/` throughout

h1: Teleport Authentication with OneLogin
---

This guide will explain how to configure [OneLogin](https://www.onelogin.com/)
to issue Teleport credentials to specific groups of users. When used in
combination with role based access control (RBAC) it allows SSH administrators
to define policies like:

- Only members of "DBA" group can connect to PostgreSQL databases.
- Developers must never SSH into production servers.
- ... and many others.

## Prerequisites

- A One Login account with admin access, and users assigned to at least two groups.
- Teleport role with access to maintaining `saml` resources.  This is available
  in the default `editor` role.

(!docs/pages/includes/commercial-prereqs-tabs.mdx!)

- (!docs/pages/includes/tctl.mdx!)

## Step 1/3. Create Teleport application in OneLogin

In the OneLogin control panel's main menu navigate to **Applications** ->
**Add App**. Using the search box, select "SAML Custom Connector (SP Shibboleth)":

![SAML Custom Connector (SP Shibboleth)](../../../img/sso/onelogin/onelogin-saml-1.png)

> Define the new Application:

Reviewer: ```suggestion
Define the new application:
```
This shouldn't be a proper noun here

![SAML Config](../../../img/sso/onelogin/onelogin-saml-1a.png)

You can find Teleport icons to upload from the following links:

- [Square Icon](../../../img/sso/onelogin/teleport.png)
- [Rectangular Icon](../../../img/sso/onelogin/teleportlogo@2x.png)

From the application's **Configuration** page, set the following values:

<Admonition type="tip">

> Set <Var name="teleport.example.com:443" description="Your Teleport proxy address and port, or cloud tenant." /> here with your Teleport Proxy address and port, or Teleport Cloud tenant (e.g. `company.teleport.sh:443`) to fill out the values below.
> </Admonition>

Reviewer: ```suggestion
Set <Var name="teleport.example.com:443" description="Your Teleport Proxy Service address and port, or cloud tenant." /> here with your Teleport Proxy Service address and port, or Teleport Enterprise Cloud tenant (e.g. `company.teleport.sh:443`) to fill out the values below.
</Admonition>
```

- **Login URL**:

>   - `https://`<Var name="teleport.example.com:443"/>`/web/login`

Author: I found that I couldn't get this connector to work unless OneLogin was given the port as well.
